### DISCUSSION TITLE
lps-node make flash error

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1695] [General]
Hi, 

I want to flash new firmware to lps-node, but make flash report error as following: 

```
bitcraze@bitcraze-vm:~/projects/lps-node-firmware$ make flash
openocd -d2 -f interface/stlink-v2.cfg  -f target/stm32f0x_stlink.cfg -c init -c targets -c "reset halt" \
           -c "flash write_image erase bin/lps-node-firmware.elf" -c "verify_image bin/lps-node-firmware.elf" -c "reset run" -c shutdown
Open On-Chip Debugger 0.11.0
Licensed under GNU GPL v2
For bug reports, read
	http://openocd.org/doc/doxygen/bugs.html
debug_level: 2

WARNING: interface/stlink-v2.cfg is deprecated, please switch to interface/stlink.cfg
embedded:startup.tcl:26: Error: Can't find target/stm32f0x_stlink.cfg
in procedure 'script' 
at file "embedded:startup.tcl", line 26
make: *** [Makefile:91: flash] Error 1

```

**Comment by ToveRumar** [comment]
Hi!

It seems like openocd has deprecated the "target/stm32f0x_stlink.cfg" file and you should use "target/stm32f0x.cfg" instead. 

Since there is a deprecation warning for interface/stlink-v2.cfg as well i would switch to using interface/stlink.cfg while you are at it.  

It looks like no one has done this in a while with a freash setup so the documentation looks to not be completely up to date. If this works for you I will update the documentation. 

Hope this works!

**Reply by justinleeyang** [reply]
Yeah，it works.

**Reply by ToveRumar** [reply]
Great, Ill update the documentation!
